app/src/main/java/ch/epfl/skysync/database/schemas/LocationSchema.kt

```diff
@@ -0,0 +1,44 @@
+package ch.epfl.skysync.database.schemas
+
+import ch.epfl.skysync.database.Schema
+import ch.epfl.skysync.models.UNSET_ID
+import ch.epfl.skysync.models.calendar.AvailabilityCalendar
+import ch.epfl.skysync.models.calendar.FlightGroupCalendar
+import ch.epfl.skysync.models.location.Location
+import ch.epfl.skysync.models.user.Admin
+import com.google.android.gms.maps.model.LatLng
+
+val UNSET_USER =
+    Admin(
+        id = UNSET_ID,
+        firstname = "",
+        lastname = "",
+        availabilities = AvailabilityCalendar(),
+        assignedFlights = FlightGroupCalendar()
+    )
+
+data class LocationSchema(
+    val id: String? = null,
+    val userId: String? = null,
+    val latitude: Double? = null,
+    val longitude: Double? = null
+) : Schema<Location> {
+    override fun toModel(): Location {
+        if (userId == null || latitude == null || longitude == null || id == null) {

```

Comment:
This is not needed, the attributes will always be not null except the corresponding attribute in the model is nullable (see `FlightSchema` for an example)

Reply:
Okay